app/models/site.rb

```diff
@@ -82,4 +89,13 @@ def pick_next_available_code(available_codes: [])
     available_undesirable_codes = available_codes & EASILY_CONFUSED_CODES
     available_desirable_codes.sample || available_undesirable_codes.sample
   end
+
+  def needs_travel_to_work_area_and_london_borough_updated?
+    latitude.present? &&
+      longitude.present? &&
+      (
+        saved_change_to_longitude? ||
```

Comment:
Should this be `travel_to_work_area.present?`, `london_borough.present?`, `saved_change_to_travel_to_work_area?` and `saved_change_to_london_borough?`

Reply:
Shouldn't this trigger when the lat/long is updated not when the attribute changes? It's not input anywhere on the UI so the only way it can be triggered is postcode update -> geolocation update -> ttwa update

Reply:
Ah yeah good point - forgot about geocoding